src/chunk_index.h

```diff
@@ -45,7 +45,7 @@ extern int ts_chunk_index_rename(Chunk *chunk, Oid chunk_indexrelid, const char
 extern int ts_chunk_index_rename_parent(Hypertable *ht, Oid hypertable_indexrelid,
 										const char *newname);
 extern int ts_chunk_index_adjust_meta(int32 chunk_id, const char *ht_index_name,
-									  const char *old_name, const char *new_name);
+									  const char *oldname, const char *newname);
```

Comment:
You renamed here the parameters and in `src/dimension.h` you made the contrary

Reply:
Switched to underscore everywhere for consistency.